docs/self-managed/operational-guides/backup-restore/_harmonized-indexes-main.md

```diff
@@ -0,0 +1,232 @@
+```mermaid
+erDiagram
+
+    camunda-authorization {
+        keyword id
+        long ownerKey
+        keyword ownerType
+        keyword resourceType
+    }
+
+    permissions {
+        keyword type
+        keyword resourceIds
+    }
+    camunda-authorization ||--o{ permissions : has
+
+    camunda-group {
+        keyword id
+        long key
+        keyword name
+        long memberKey
+        join join
+    }
+    camunda-group ||--o{ camunda-group: "group:member"
```

Comment:
I forgot to mention this but I snuck these `join` treatments into the generated markdown manually, to see what they would look like. The scripts would need to be updated to accommodate these, if we decide we like them.